test/berlim/accounts/accounts_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Berlim.Accounts.AccountsTest do
+  use Berlim.DataCase
+
+  import Berlim.Factory
+  import Comeonin.Bcrypt, only: [hashpwsalt: 1]
+
+  alias Berlim.Accounts
+
+  test "token_sign_in/2 with valid admin credentials" do
+    admin = insert(:admin, %{email: "john@email.com", encrypted_password: hashpwsalt("123456")})
+
+    {:ok, token, claims} = Accounts.token_sign_in("john@email.com", "123456")
+
+    assert token
```

Comment:
qual sua intenção aqui?

Reply:
testar que vem um token

Reply:
esse assert só vai falhar se token for `nil` ou `nil`, se quer garantir que vem um token seria bom fazer `assert is_binary(token)`